MANIFEST.in

```diff
@@ -6,5 +6,5 @@ include README.md
 recursive-include tests *
 recursive-exclude * __pycache__
 recursive-exclude * *.py[co]
-
+recursive-include *.txt
```

Comment:
I'm not actually all that sure, to be honest. I've had varying behavior with `MANIFEST.in` files in the past, so I concluded to just stay safe by always being more explicit with extensions than not.